### PR TITLE
Problem of the operation of the _transformDirty & _additionalTransformDirty flag.

### PR DESCRIPTION
**Pull request by yutaka-takeda-drecom**
There is a problem of the operation of the _transformDirty & _additionalTransformDirty flag. 
This problem occurred in a change from 3.11 to 3.12.
The fix for CCNode of [this committed](https://github.com/cocos2d/cocos2d-x/commit/69d2ac1786d29dd519bd9692447896cddff09f02), there is a change in the flag of the operation.
(The name of the flag is changed from _useAdditionalTransform at 3.12 _additionalTransformDirty)

Therefore , would wrong to update the transform.

### CODE REVIEW

**Comment by minggo** [comment]
@ricardoquesada could you please take a look?

**Comment by ricardoquesada** [comment]
@yutaka-takeda-drecom could you describe in more detail the issue? what is the best way to reproduce it? thanks.

**Comment by ricardoquesada** [comment]
[ci rebuild]

**Comment by ricardoquesada** [comment]
[ci rebuild]

File: cocos/2d/CCNode.cpp [hunk 1]
```diff
@@ -1741,25 +1741,25 @@ const Mat4& Node::getNodeToParentTransform() const
...
+            // is being overwritten everyframe, _additionalTransform[1] is used to have a copy
+            // of the last "_transform without _additionalTransform"
+            if (_transformDirty)
+                _additionalTransform[1] = _transform;
+            
+            if (_transformUpdated)
+                _transform = _additionalTransform[1] * _additionalTransform[0];
+        }
+        
+        _transformDirty = false;
```
Comment (ricardoquesada) [inline cocos/2d/CCNode.cpp:1760]: why `_additionalTransformDirty = false;` was removed? 
isn't it needed anymore here? thanks.

**Comment by halx99** [comment]
[ci rebuild]

**Comment by yutaka-takeda-drecom** [comment]
I am sorry.
I have found that there is a misunderstanding.
This pull request is close.

**Pull request closed without merging**